### PR TITLE
Astaghfirullah, I am introducing pork

### PR DESCRIPTION
**Pull request by ThePainkiller**
## For he is merciful and will surely forgive me.

- Makes pigs (and most porcine animals) drop more than just reskinned meat: actual pork chops to cook with! When sliced on a table, they produce raw strips of bacon that are also used in some recipes.
image
- Reworked some recipes to include pork and bacon, namely: Bacon, Tonkatsu, Dumplings, Sausage.
- Fried bacon now done with 1 raw bacon instead of cutlet. Dumplings require two raw bacon strips instead of cutlets. Tonkatsu requires 1 raw pork slab instead of a cutlet. Sausage requires 1 raw bacon instead of cutlet.
- Added glazed pork chops, in a microwave insert 1 raw porcine meat, 5 units of Barbecue sauce, 1 unit salt, 1 unit pepper
- Adds Vinegar and Barbecue sauce. Vinegar is made with equal parts wine, flour and water. Barbecue sauce with equal parts ketchup, vinegar and sugar.
- New condiment bottle and packet sprites for vinegar and bbq sauce
image
- Corrects protein amount on recipes that included meat products as they gave less protein than its raw counterparts, making cooking food have actual protein loss across the board.
- Adds garlic bread sprite, currently unused as there's no garlic in game yet

## Changelog
:cl:
add: Adds proper pork meat with its unique sprite, makes all porcine animals drop it on butchering. Cutting it on a table yields raw bacon strips. Also adds glazed pork chops, made with 1 raw pork meat, 5u barbecue sauce, 1u salt and 1u pepper
add: Adds Vinegar (equal parts wine, flour and water) and Barbecue sauce (equal parts ketchup, sugar and vinegar) as condiments
add: Dinnerware express starts with packs of BBQ sauce and vinegar now
tweak: Replaces beef for pork on the following recipes:
Tonkatsu uses 1 raw pork meat instead of a cutlet
Dumplings need 2 pieces of raw bacon instead of cutlets
Sausage needs 1 raw bacon instead of a cutlet
Bacon is made with 1 raw bacon instead of a cutlet now
balance: Chef gets +10 BIO instead of +5 VIG to better help with butchering meat. Bacon needs 1u salt instead of 5u.
fix: fixes some meat based recipes actually having less protein than their uncooked parent meats, makes cooking food actually better in terms of overall nutrition instead of worse.
imageadd: Adds garlic bread sprite, unused for now. Adds vinegar and BBQ sauce condiment bottles and packs sprites.
refactor: moves bacon to food_ingredients.dmi
/:cl:

### CODE REVIEW

File: code/modules/reagents/recipes/recipes_food_drinks.dm [hunk 1]
```diff
@@ -96,6 +96,15 @@
 	required_reagents = list("tomatojuice" = 2, "water" = 1, "sugar" = 1)
 	result_amount = 4
 
+/datum/chemical_reaction/vinegar
+	result = "vinegar"
+	required_reagents = list("wine" = 1, "water" = 1, "flour" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/barbecue
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/recipes/recipes_food_drinks.dm:104]: No salts?

Comment (ThePainkiller) [reply code/modules/reagents/recipes/recipes_food_drinks.dm:104]: Making vinegar involves a "mother base" (I'm using flour as a substitute for yeast/vinegar mother/etc), wine, and all dilluted in a bit of water, it doesn't take salt to make!

**Comment by cdb-is-not-good** [comment]
full tested and working?

**Comment by MSieghardt** [comment]
Looks fine, nothing much that can brick the server. I'd say test to see if you can make every changed recipe and we should be fine.

**Comment by ThePainkiller** [comment]
Had my reservations on water and flour but I remembered dough here is made with eggs so it shouldn't conflict with any other recipes.
I'll localtest when I get home just to be safe, and will post results.

**Comment by ThePainkiller** [comment]
Fixed a whole bunch of problems now, merges and works well, all of the recipes involved are done, good to go unless Linter chan hates me.

**Pull request merged**